README.md

```diff
@@ -56,3 +56,33 @@ Cordova_in_app_browser.ai_clear_cache true] in
 (* Opens in the Cordova WebView if the URL is in the white list *)
 i#open_ "https://ocaml.org" (Cordova_in_app_browser.target_self) opt
 ```
+
+The function "open_t" is hte same as "open" but it return a "typed"
```

Comment:
the

Reply:
See this pull-request for the correction and if others corrections are needed for the README:
https://github.com/besport/ocaml-cordova-plugin-inappbrowser/pull/3

Reply:
(@dannywillems No, it's not yet ready to be reviewed yet.)